### DISCUSSION TITLE
这周你做了什么？ 2022/03/28 - 2022/04/03

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/emacstalk.github.io#10] [Show and tell]
- 可以随便讨论本周你打算做什么与 Emacs 相关的事情，甚至可以寻求帮助或反馈。
- 请记住，什么都不做也没关系！
- Happy Hacking Emacs！

**Comment by douyipu** [comment]
最近又拾起emacs，这周把TUTORIAL文档又看了一遍。以前一直用typora写markdown，这周开始emacs和typera混用。emacs的markdown的ilist也还行。

**Comment by VSteveHL** [comment]
准备装个 Linux，把配置文件改成 Windows 和 Linux 都能用的。

**Comment by qiandu2006** [comment]
尝试完全在spacemacs下工作，而不是时不时切换回vscode

**Reply by jiacai2050** [reply]
> 尝试完全在spacemacs下工作，而不是时不时切换回vscode

好奇是什么场景下需要切回到 VSCode？

**Reply by qiandu2006** [reply]
主要还是对emacs不够熟，没有打造出舒适的定制环境

**Comment by lakkiy** [comment]
用内置的 xwidget + webjump 很舒服， advice 一下 xwidget 让它在固定的 tab 打开。

删掉了 org-roam 和 org-roam ui，待了很长时间了发现用的不多，反而那种拿起来就随便乱记的比较适合我，所以上车 xeft

**Reply by lakkiy** [reply]
写成了上周干什么了 🥵

**Reply by jiacai2050** [reply]
哈哈，这是第一次开帖，可以记录下之前的。
